code/controllers/subsystem/movement/movement_types.dm

```diff
@@ -42,6 +42,11 @@
 	src.lifetime = timeout
 	return TRUE
 
+/datum/move_loop/proc/compare_loops(datum/move_loop/loop_type, priority, flags, extra_info, delay = 1, timeout = INFINITY) //exists so we can check if this exact moveloop datum already exists (in terms of vars) and so we can stop it from needlessly create a new one to overwrite
```

Comment:
You should make this comment a doc comment (assuming bee has those)

Reply:
bee doesn't trough i can see it beeing a bit clearer to see if i use it with that format (in contrast to same line)